mico-core/src/main/java/io/github/ust/mico/core/dto/MicoServiceStatusDTO.java

```diff
@@ -40,38 +46,147 @@
+        }
+    )})
     private String name;
 
     /**
-     * shortName of a {@link io.github.ust.mico.core.model.MicoService}.
+     * ShortName of a {@link MicoService}.
      */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "ShortName"),
+            @ExtensionProperty(name = "x-order", value = "10"),
+            @ExtensionProperty(name = "description", value = "Short name of a MicoService.")
+        }
+    )})
     private String shortName;
 
     /**
-     * Version of a {@link io.github.ust.mico.core.model.MicoService}.
+     * Version of a {@link MicoService}.
      */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "Version"),
+            @ExtensionProperty(name = "x-order", value = "20"),
+            @ExtensionProperty(name = "description", value = "Version of a MicoService.")
+        }
+    )})
     private String version;
 
     /**
-     * Counter for number of replicas that should be available.
+     * Counter for the number of replicas that should be available.
      */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "RequestedReplicas"),
+            @ExtensionProperty(name = "x-order", value = "40"),
+            @ExtensionProperty(name = "description", value = "Counter for number of replicas of a MicoService that should be available.")
+        }
+    )})
     private int requestedReplicas;
 
     /**
-     * Counter for that are actually available.
-      */
+     * Counter for the number of replicas that are actually available.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "AvailableReplicas"),
+            @ExtensionProperty(name = "x-order", value = "50"),
+            @ExtensionProperty(name = "description", value = "Counter for replicas of a MicoService that are actually available.")
+        }
+    )})
     private int availableReplicas;
 
     /**
      * Each item in this list represents a Kubernetes Service.
      */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "InterfacesInformation"),
+            @ExtensionProperty(name = "x-order", value = "60"),
+            @ExtensionProperty(name = "description", value = "Each item in this list represents a Kubernetes Service.")
```

Comment:
Make clear in the comment and in the description that this list of Kubernetes services are the interfaces of this MicoService